typings/bson/json_util.pyi

```diff
@@ -1,4 +1,5 @@
 from typing import Any
 
 def loads(s: str) -> Any: ...
-def dumps(obj: object) -> str: ...
+def dumps(obj: object, *args, **kwargs) -> str: ...
```

Comment:
```suggestion
def dumps(obj: object, *args: Any, **kwargs: Any) -> str: ...
```

Reply:
Does that make sense? In both cases, you can pass any variable